frontend/view-edit-client/payments/create-edit/edit-payment-info.component.tsx

```diff
@@ -1,6 +1,9 @@
 import React from "react";
 import css from "./edit-payment-info.css";
 import { CreatePaymentStepProps } from "../create-payment.component";
+import MultiClientSelect, {
```

Comment:
Let's leave payments-related changes for a different PR. Here are some git commands you can run to undo the changes on this branch and move them to a different branch.

```ts
git checkout master

# Create a new branch based off of master
git checkout -b multiple-clients-payments

# Get the payment files from the multiple-clients branch into the multiple-clients-payments branch
git checkout multiple-clients -- frontend/view-edit-client/payments/create-edit/edit-payment-info.component.tsx frontend/view-edit-client/payments/create-payment.component.tsx frontend/view-edit-client/payments/edit-payment.component.tsx

# You should now have the payment changes on this new branch
git status
git add .
git commit -m "Beginnings of multiple clients on payments"

# Go back to the invoice multiple-clients branch
git checkout multiple-clients
# undo changes made to payments files
git checkout master -- frontend/view-edit-client/payments/create-edit/edit-payment-info.component.tsx frontend/view-edit-client/payments/create-payment.component.tsx frontend/view-edit-client/payments/edit-payment.component.tsx
# verify it says that those files have changed
git status
git add .
git commit -m "Undoing payment changes"
git push
```

In the future, you should avoid getting into this situation by creating a new branch when starting to work on a new feature. The branch can be based off of your existing branch, or off of master. The best one to practice is always branching off of master, since branching of your existing branch requires more advanced git skill.

Reply:
Gotcha, fixed it and followed the commands. Had to do --no-verify for the payments branch since the master doesn't have the multi-clients and single client components in it. Hopefully that can be fixed when this PR is merged to the master